### DISCUSSION TITLE
Typesense queries without running machine-learning server

### DISCUSSION THREAD

**Discussion by martin5233** [immich-app/immich#5159] [Q&A]
I have set up my Immich instance to use my notebook as its machine-learning server, while all other services are located on a smaller and slower PC running 24/7. I thought, that the machine-learning server was only required for the actual learning part, i.e. classifying faces and extracting data for TypeSense searches. Now I noticed, that even the searches do not work, when my notebook is turned off. Would it be possible to run the queries on the main server somehow?

**Comment by schuhbacca** [comment]
I think quite a few people switch back to the low power server for machine learning once the initial large ingest is completed.

**Comment by martin5233** [comment]
That's what I thought about as well. But I was also interested, if this would be possible, because from my understanding the retrieval of this information should be less resource-intensive than the ingestion.

**Comment by aviv926** [comment]
As far as I know, after Immich has done the hard initial work, it will only require 1% of the CPU, which means that for each image that you upload, the computer will indeed activate all the tools of the ML, but it will not require that much processing power
You can read more about it here:
https://discord.com/channels/979116623879368755/994044917355663450/1174918629398151188

I think even a 

 > smaller and slower PC running 24/7

 will be able to handle it
